kitchen.yml

```diff
@@ -9,3 +9,9 @@ suites:
   - name: homes
     run_list:
       - recipe[osl-nfs::homes]
+
+provisioner:
+  name: chef_solo
+  #enforce_idempotency: true
+  multiple_converge: 2
```

Comment:
Let's comment this out as well since we don't need to have it run twice. Also please add a comment about why this is failing.